### PR TITLE
Changed to new AAS package

### PR DESCRIPTION
**Pull request by SergioCasCeb**
Changed from the Node-wot td-tools to the new Thing Web AAS package.

### CODE REVIEW

File: packages/web/webpack.config.js [hunk 1]
```diff
@@ -53,8 +53,15 @@ module.exports = (env, argv) => {
             hot: false,
             compress: true,
             historyApiFallback: true,
-        } 
-        : {},
```
Comment (egekorkan) [inline packages/web/webpack.config.js:57]: why is this change necessary?

File: packages/web/src/scripts/util.js [hunk 2]
```diff
@@ -25,7 +25,8 @@ import { convertTDJsonToYaml, convertTDYamlToJson, tdValidator, tmValidator, com
 import tdToOpenAPI from '../../../td_to_openAPI/dist/web-bundle.min.js'
 import tdToAsyncAPI from '../../../td_to_asyncapi/dist/web-bundle.min.js'
 import { addDefaults, removeDefaults } from '../../../defaults/dist/web-bundle.min.js'
-import { AssetInterfaceDescriptionUtil } from '@node-wot/td-tools/dist/util/asset-interface-description.js'
+// import { AssetInterfaceDescriptionUtil } from '@node-wot/td-tools/dist/util/asset-interface-description.js'
```
Comment (egekorkan) [inline packages/web/src/scripts/util.js:28]: comment can be removed

**Review by egekorkan — CHANGES_REQUESTED** [review]

File: packages/web/src/scripts/util.js [hunk 3]
```diff
@@ -166,13 +167,13 @@ export function generateAAP(fileType, editorInstance) {
...
 export function generateAAS(fileType, editorInstance) {
-    const assetInterfaceDescriptionUtil = new AssetInterfaceDescriptionUtil()
+    const assetInterfaceDescription = new AssetInterfacesDescription()
 
     const tdToConvert = fileType === "json"
         ? editorInstance.getValue()
         : convertTDYamlToJson(editorInstance.getValue())
 
-    const AASInstance = assetInterfaceDescriptionUtil.transformTD2SM(tdToConvert)
+    const AASInstance = assetInterfaceDescription.transformTD2AID(tdToConvert, { createAAS: false }, [])
```
Comment (danielpeintner) [inline packages/web/src/scripts/util.js:176]:
```suggestion
    const AASInstance = assetInterfaceDescription.transformTD2AID(tdToConvert)
```

You should be able to simplify it.

File: packages/web/webpack.config.js (hunk 1 above)
Comment (SergioCasCeb) [reply packages/web/webpack.config.js:57]: I belive that was just a little format fix that occurred when I was formating the document

Comment (egekorkan) [reply packages/web/webpack.config.js:57]: but you are adding a resolve object?

Comment (SergioCasCeb) [reply packages/web/webpack.config.js:57]: Yeah since that specific configuration should only exist when building the application in developer mode. So when the project is built in production, there shouldn't be any specific configuration for the development server.

Comment (SergioCasCeb) [reply packages/web/webpack.config.js:57]: Also, this is not something that was added now, it has been there since the beginning. Just the formatting of the file caused it to move.
